images/plbase/python-requirements.txt

```
@@ -31,17 +31,17 @@ pytest-repeat==0.9.1
 pytest==7.4.0
 recommonmark==0.7.1
 regex==2023.8.8
-rpy2==3.5.12
+rpy2==3.5.13
```

```suggestion
rpy2==3.5.12
```
3.5.13 is failing to build, both in CI and locally. Restoring as 3.5.12 builds successfully.